src/frontend/components/ViewersList/components/ViewersListHeader/index.tsx

```diff
@@ -1,26 +1,21 @@
 import React from 'react';
 import { Box, Text } from 'grommet';
+import { MarginType } from 'grommet/utils';
 import styled from 'styled-components';
 
 const StyledTextHeader = styled(Text)`
   font-family: 'Roboto-Regular';
   letter-spacing: -0.2px;
 `;
 
-interface StudentViewersListHeaderProps {
+interface ViewersListHeaderProps {
```

Comment:
I would rather exdends BoxProps from grommet here (so you can override all props from Box component and not only those you add manually)

Reply:
I don't want to be able to set anything else than the margin, for this component

Reply:
done in : [15664879](https://github.com/openfun/marsha/pull/1443/commits/15664879e367d24b957f09215e2eb3f64cd91fa1)